bndtools.core/src/bndtools/central/Central.java

```diff
@@ -240,9 +240,10 @@ public static Workspace getWorkspace() throws Exception {
 				// one
 				File workspaceDirectory = getWorkspaceDirectory();
 				// Check to see if we need to convert it...
-				if (workspaceDirectory != null && ws.isDefaultWorkspace()) {
```

Comment:
You should pull this commit out into its own PR so it does not languish here...

Reply:
Done; see #4582.